compiler/src/iree/compiler/Codegen/Common/GPU/GPUNestedLayoutDistributionPatterns.cpp

```diff
@@ -240,6 +240,36 @@ static VectorValue getSlicedPermutedMask(PatternRewriter &rewriter,
   return slicedMask;
 }
 
+/// Project a vector based on a provided projection map.
+/// Firstly, this will tranpose the vector in a way sliced out
+/// dims become outermost. Then it performs a vector.extract
+/// remove the dims that are not present in the results of the map.
+static VectorValue projectVector(RewriterBase &rewriter, Location loc,
+                                 VectorValue val, AffineMap projectionMap) {
+  llvm::SmallVector<int64_t> remaningDims;
+  SmallVector<int64_t> allDims =
+      llvm::to_vector(llvm::seq<int64_t>(projectionMap.getNumDims()));
+  llvm::SmallDenseSet<int64_t> slicedDims{allDims.begin(), allDims.end()};
```

Comment:
style nit: avoid using braced initializer lists to call a constructor.

```suggestion
  llvm::SmallDenseSet<int64_t> slicedDims(allDims.begin(), allDims.end());
```

https://llvm.org/docs/CodingStandards.html#do-not-use-braced-initializer-lists-to-call-a-constructor

The style guide does not explain the reason well. There is an article from abseil, which mentions few benefits. E.g., we can avoid the most vexing parse, etc. See https://abseil.io/tips/88 for more details.

Reply:
oh thanks! now i know. done.